docs/real-world-examples/ccfraud.md

```diff
@@ -0,0 +1,50 @@
+
+# Getting started
+
+### Provisioning
+Please follow the instructions in the [quickstart](../quickstart.md) to provision an open sandbox. Make note of the name of the resource group you provisioned, as well as the name of the workspace.
+
+### Kaggle credentials
+In the resource group, provisioned in the previous step we need create two secrets in order for Kaggle API to be able to download the dataset. First we need to assign current user the access policies to be able to create secrets:
+- Open "Access Policies" tab in the newly created keyvault
+- Click "Create" button in the top
+- Select "Select all" right under "Secret Management Operations" and press "Next"
+- Lookup currently logged in user (using user id or an email), select it and press "Next"
+- Press "Next" and "Create" in the next screens
+
+Afterwards we are able to create the secrets. Start by opening "Secrets" tab. Here we will need to create two plain text secrets with following names and contents:
+- **kaggleusername** - specifies your kaggle user name
+- **kagglekey** - this is API key that can be obtained from your profile on the kaggle
+
+### Data provisioning
+Before we dive deep into training the model we need to upload the data from Kaggle dataset into the silo datastores in corresponding geolocations. This can all be performed with ease using **data provisioning pipeline**. To run it follow these steps:
+
+- Navigate to: `examples/pipelines/utils/upload_data/` folder
+- Run `submit.py --submit` command
+
+:warning: Proceed to the next step only once the pipeline finishes.
+
+### Model choice
+Please update [**config**](../../examples/pipelines/ccfraud/config.yaml), field `model_name` in the `training_parameters` section, to reflect desired model to be trained, options include: SimpleLinear, SimpleLSTM, SimpleVAE
+
+- **SimpleLinear** - model fully composed of `torch.Linear` layers with `ReLU` activations, takes data as-is sample-by-sample
+- **SimpleLSTM** - model composed by 4 LSTM layers connected to linear head with architecture similar to **SimpleLinear**, takes data ordered by time in sequences that overlap each other
+- **SimpleVAE** - model composed of 2 encoder LSTM layers and 2 decoder LSTM layers that tries to recreate consumed sequence of transactions, the latent space created by encoder is consumed by a linear layer to perform prediction, takes data ordered by time in sequences that overlap each other
+
+## Running example
+- Update [**config**](../../examples/pipelines/ccfraud/config.yaml) to reflect your AML FL orchestration setup
```

Comment:
Here might be good to add a simple sentence explaining a bit more what needs to be updated/adjusted, e.g., in Amit/Thomas PRs, they added this "(if you kept everything default you'll only have to adjust subscription id, resource group, and workspace name)". Might be good to add sth similar here for easier consumption by the customers.